peodd/peodd.py

```diff
@@ -28,20 +28,24 @@
 
 import click
 import tomli
-
 from release_tools.project import Project
-from release_tools.semverup import find_pyproject_file
 from release_tools.repo import RepositoryError
+from release_tools.semverup import find_pyproject_file
 
 VERSION_NUMBER_REGEX = r"\d+(?:\.\d+)+"
 
 
 @click.command()
 @click.option('-o', '--output',
-              default="requirements-dev.txt",
```

Comment:
I would remove the `default` argument as a whole but keep it as a required argument, so that the users need to provide the file name.

Reply:
```python
@click.command()
@click.option('-o', '--output',
              show_default=True,
              help="Output filename for the dependencies")
@click.option('-n', '--not-dev',
              default=False,
              is_flag=True,
              show_default=True,
              help="Output not-dev dependencies")
```

Reply:
Since we are not having any default option, I think we can remove the `show_default=True,` too.
WDYT?

Reply:
Of course! awkward, haha!